tasks/collect_files_to_commit.js

```diff
@@ -0,0 +1,15 @@
+import filesToCommit from './utils/files_to_commit';
+import { resolve } from 'path';
+
+const root = resolve(__dirname, '..');
+
+export default function registerCollectFilesToCommit(grunt) {
+  grunt.registerTask('collectFilesToCommit', function () {
+    filesToCommit(root)
```

Comment:
yeah, there are a few paths defined in https://github.com/elastic/kibana/blob/a42f29fe237a0026e90376e4bc409ba9a5651ec2/Gruntfile.js